src/unix/aix.c

```diff
@@ -760,16 +760,15 @@ static void uv__ahafs_event(uv_loop_t* loop, uv__io_t* event_watch, unsigned int
       p++;
     }
   } else {
-    /* For file changes, figure out whether filename is absolute or not */
-    if (handle->path[0] == '/') {
-      p = strrchr(handle->path, '/');
+    p = strrchr(handle->path, '/');
+    if (p == NULL)
+      p = handle->path;
+    else
+      p++;
+    while (*p != NULL) {
+      fname[i]= *p;
```

Comment:
Sounds good.  Maybe make that `strncpy(fname, handle->path, sizeof(fname) - 1); fname[sizeof(fname) - 1] = '\0';`, just in case.